editor/sidebar/last-revision/index.js

```diff
@@ -81,6 +81,10 @@ class LastRevision extends Component {
 		const { revisions } = this.state;
 		const lastRevision = revisions.length ? revisions[ 0 ] : null;
 
+		if ( revisions.length <= 0 ) {
```

Comment:
Would it ever be negative? Maybe `! revisions.length` ? Or at least `revisions.length === 0` to more accurately describe what we're anticipating.